templates/component.yaml

```diff
@@ -0,0 +1,93 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+component-definition:
+  components:
+    - control-implementations:
+        - description: Control Implementation Description
+          implemented-requirements:
+            - control-id: ac-6.9
+              description: <how the specified control may be implemented if the containing component or capability is instantiated in a system security plan>
+              remarks: |-
+                ASSESSMENT-OBJECTIVE:
+                the execution of privileged functions is logged.
+              uuid: {{ uuid }}
+            - control-id: au-12
+              description: <how the specified control may be implemented if the containing component or capability is instantiated in a system security plan>
+              remarks: |
+                ASSESSMENT-OBJECTIVE:
+                AU-12a. audit record generation capability for the event types the system is capable of auditing (defined in AU-02_ODP[01]) is provided by [Assignment: organization-defined system components];
+                AU-12b.  [Assignment: organization-defined personnel or roles] is/are allowed to select the event types that are to be logged by specific components of the system;
+                AU-12c. audit records for the event types defined in AU-02_ODP[02] that include the audit record content defined in AU-03 are generated.
+              uuid: {{ uuid }}
```

Comment:
do we want to make these controls a set we expect most apps to answer if they say met the silver standard and integrated with Istio/Prometheus/Loki?